sunpycube/cube/NDCube.py

```diff
@@ -71,12 +69,8 @@ def plot(self, *args, **kwargs):
     def __getitem__(self, item):
         if item is None or (isinstance(item, tuple) and None in item):
             raise IndexError("None indices not supported")
-        return self._getitem_NDCube(item)
-
-    def _getitem_NDCube(self, item):
         data = self.data[item]
-        # wcs = self.wcs.wcs_slicer[item]
-        wcs = None
+        wcs = self.wcs.wcs_slicer[item]
```

Comment:
I think (for reasons of WCS classes and API) that it would be better if this was `wcs = slice_wcs(self.wcs, item)`.

I realise this is the whole "passing slices to functions" thing, but it's also not public API.